scripts/apps/web-publisher/controllers/WebPublisherManagerController.js

```diff
@@ -260,14 +275,105 @@ export function WebPublisherManagerController($scope, publisher, modal) {
 
         /**
          * @ngdoc method
+         * @name WebPublisherManagerController#toogleCreateListCard
+         * @param {String} listType - type of content list
+         * @description Creates a new unsaved content list card
```

Comment:
Y, right, I changed logic, so now its not toogle, but forgot to change name, so I'll remove prefix.